zyte_common_items/pipelines.py

```diff
@@ -53,3 +53,82 @@ def __init__(self):
 
     def process_item(self, item, spider):
         return ae.downgrade(item)
+
+
+class DropLowProbabilityItemPipeline:
+    """This pipeline drops an item if its probability, defined in the settings,
+    is less than the specified threshold.
+
+    There are two ways that the thresholds can be defined per item type using
```

Comment:
These approaches can also be mixed. I think we can have a single example, something like this:

```py
from zyte_common_items import Article

ITEM_PROBABILITY_THRESHOLDS = {
    Article: 0.2,
    "zyte_common_items.items.Product": 0.3,
    "default": 0.15,  # if not provided, 0.1 is used
}
```